cpp/cmake/modules/FindUFCx.cmake

```diff
@@ -48,6 +54,8 @@ execute_process(
     "import ffcx.codegeneration, sys; sys.stdout.write(ffcx.codegeneration.get_include_path())"
   OUTPUT_VARIABLE UFCX_INCLUDE_DIR
 )
+# TODO: CMake 3.20 has more modern cmake_path.
```

Comment:
Let's update to CMake 3.21 (see above comment).

Reply:
See comment on Rocky 9.